cmd/buildah/commit.go

```diff
@@ -123,11 +132,17 @@ func commitCmd(c *cli.Context) error {
 		dest = dest2
 	}
 
+	systemContext, err := systemContextFromOptions(c)
+	if err != nil {
+		return errors.Errorf("error building system context [%v]", err)
```

Comment:
Change this to return `errors.Wrapf(err, "error building system context")`.

Reply:
Fixed